lib/auth/apiserver.go

```diff
@@ -150,9 +150,7 @@ func NewAPIServer(config *APIConfig) (http.Handler, error) {
 	srv.DELETE("/:version/tunnelconnections/:cluster", srv.withAuth(srv.deleteTunnelConnections))
 	srv.DELETE("/:version/tunnelconnections", srv.withAuth(srv.deleteAllTunnelConnections))
 
-	// Server Credentials
-	srv.POST("/:version/server/credentials", srv.withAuth(srv.generateHostCerts))
```

Comment:
Can v8 clients call this method? If we're removing it from client side in v9, shouldn't we keep the server API until v10?

Reply:
v8 clients will try GRPC first, and only attempt to call this if GRPC is not implemented. v8 servers have the GRPC, so the only way a v8 client could call this if if they hit a v7 server (which we don't allow, as you have to upgrade proxy/auth before nodes).